adapter-s4j/src/main/resources/sanity-validation/utilities.sh

```diff
@@ -0,0 +1,113 @@
+#! /usr/local/bin/bash
```

Comment:
same concern as with the other shell code. Essentially, this is an opaque type which nosqlbench doesn't understand and can't verify, validate, use directly, or marshal. It is surely a security vector.

Reply:
Deleted the bash scripts (as per our discussion)